tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_dm_ops.cpp

```diff
@@ -534,6 +536,28 @@ namespace tt::tt_metal::detail{
             py::arg("input_tensors"), py::arg("dim"), py::arg("num_links") = 1, py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
             R"doc(Performs all gather on a list of tensors that form one tensor that is distributed across devices. The output is a list of a tensor which has been duplciated across the input devices.)doc"
         );
+
+        // Reduce Scatter
+        m_tensor.def("reduce_scatter", &reduce_scatter,
+            // py::arg("input_tensors"), py::arg("scatter_split_dim"), py::arg("reduce_op") = ReduceOpMath::SUM, py::arg("num_links") = 1, py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
```

Comment:
Remove?

Reply:
Resolved although clang format was required to run so the whole file is now modified :(

Reply:
If you want then maybe just skip the precommit check when you remove the comment so we can avoid formatting it?

Reply:
okay I ended up doing that and squashing in